src/steps/pipeline/index.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+} from '@jupiterone/integration-sdk-core';
+
+import { createAPIClient } from '../../client';
+import { IntegrationConfig } from '../../config';
+
+import { Entities, Steps } from '../constants';
+import { createPipelineEntity } from './converter';
+
+export async function fetchPipelines({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const apiClient = createAPIClient(instance.config);
+
+  await jobState.iterateEntities(
+    { _type: Entities.USER_GROUP._type },
+    async (userGroupEntity) => {
+      if (userGroupEntity.slug) {
+        await apiClient.iteratePipelines(
+          userGroupEntity.slug.toString(),
+          async (pipeline) => {
+            const pipelineEntity = createPipelineEntity(pipeline);
+            await jobState.addEntity(pipelineEntity);
+          },
+        );
+      }
+    },
+  );
+}
+
+export const pipelinesSteps: IntegrationStep<IntegrationConfig>[] = [
+  {
+    id: Steps.PIPELINES,
+    name: 'Fetch Pipelines',
+    entities: [Entities.CONFIGURATION],
```

Comment:
I'd call this pipeline to match with the others. The class of `Configuration` is good, but its name should represent the resource name.

```suggestion
    entities: [Entities.PIPELINE],
```